ansible_rulebook/builtin.py

```diff
@@ -817,6 +817,7 @@ async def event_callback(event: dict) -> None:
             rule_uuid=source_rule_uuid,
             status=controller_job["status"],
             run_at=controller_job["created"],
+            job_url=_controller_job_url(controller_job),
```

Comment:
@bzwei In the server side its going into a column called url, so maybe instead of it being specific it should just be url. Whatever that URL points to the UI will try to open it.